sdk/storage/storage-common/package.json

```diff
@@ -0,0 +1,112 @@
+{
+  "name": "@azure/storage-common",
+  "sdk-type": "client",
+  "sideEffects": false,
+  "private": true,
+  "author": "Microsoft Corporation",
+  "version": "1.0.0",
+  "description": "Azure Storage Common Client Library for JavaScript",
+  "license": "MIT",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/storage-internal-avro/",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "keywords": [
+    "azure",
+    "cloud"
+  ],
+  "browser": "./dist/browser/index.js",
+  "types": "./dist/commonjs/index.d.ts",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "files": [
+    "dist/",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
+    "build:samples": "echo Skipped.",
+    "build:test": "echo Obsolete.",
+    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "dev-tool run vendored rimraf --glob dist dist-* types temp statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
+    "docs": "echo Skipped.",
+    "extract-api": "dev-tool run build-package && dev-tool run extract-api",
+    "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "dev-tool run test:vitest --no-test-proxy --esm",
+    "lint": "eslint package.json api-extractor.json src test",
+    "lint:fix": "eslint package.json api-extractor.json src test --fix",
+    "pack": "npm pack 2>&1",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "dev-tool run test:vitest --no-test-proxy",
+    "update-snippets": "dev-tool run update-snippets"
+  },
+  "dependencies": {
+    "@azure/core-rest-pipeline": "^1.19.1",
+    "events": "^3.3.0",
+    "stream": "^0.0.3",
```

Comment:
Is this needed?

Reply:
I don't see it being in any of the package.json files under the storage folder in the main branch.

Reply:
Already removed streams.  Events is needed, but not streams